Rakefile.rb

```diff
@@ -68,6 +66,42 @@
   end
 end
 
+def prepare_and_execute_build(integration, dont_error_on_build: false)
+  sh "cd #{PROJECT_DIR} && bundle update"
+  puts "building integration #{integration}"
+
+  manifest = JSON.parse(File.read("/#{ENV['INTEGRATIONS_REPO']}/#{integration}/manifest.json"))
+  # The manifest should always have a version
+  integration_version = manifest['version']
+  if linux?
+    manifest['supported_os'].include?('linux') || return
+  elsif windows?
+    manifest['supported_os'].include?('windows') || return
+  elsif osx?
+    manifest['supported_os'].include?('osx') || return
+  end
+
+  header = erb_header({
+    'name' => "#{integration}",
+    'version' => "#{integration_version}",
+    'build_iteration' => "#{ENV['BUILD_ITERATION']}",
+    'integrations_repo' => "#{ENV['INTEGRATIONS_REPO']}"
+  })
+
+  header = erb_header({
```

Comment:
you don't seem to be using the header's anywhere currently? Am I missing something? I was expecting something like:
```
sh "(echo '#{header}' && cat #{PROJECT_DIR}/resources/datadog-integrations/software.rb.erb) | erb > #{PROJECT_DIR}/config/software/dd-check-#{ENV['INTEGRATION']}-software.rb"
```